libs/extensions/std/lang/src/text-file-interpreter-meta-inf.ts

```diff
@@ -25,18 +26,16 @@ export class TextFileInterpreterMetaInformation extends BlockMetaInformation {
           },
           validation: (property, context) => {
             const propertyValue = property.value;
-            assert(isTextLiteral(propertyValue));
+            assert(isExpression(propertyValue));
```

Comment:
This seems to be repeated quite a bit, might be good to have a util method to get property values of a specific type. I.e. `evaluatedPropertyExpression<T>` (in this case `evaluatedPropertyExpression<string>` that reads the property, evaluates the expression, asserts the type and returns the correctly typed result.

Reply:
https://github.com/jvalue/jayvee/pull/290/commits/4c63e1dbfd4d90495c741b04c3dffcd9662d79ff